### ISSUE TITLE
Invalid player data

### ISSUE THREAD

**Issue by 5A93** [DakotaPride/vanilla-claws-forge#3]
version 1.19.2-1.1. being refused to join the world after adding this mod, it said invalid player data. Perhaps incompatible with Epic Fight or something?

**Comment by DakotaPride** [comment]
Have you attempted to join the world with just Vanilla Claws (and any required dependencies) installed? Do you have a latest.log or similar file that could provide any detailed errors?

**Comment by 5A93** [comment]
> 您是否尝试过仅安装 Vanilla Claws（以及任何必需的依赖项）就可以加入这个世界？您是否有可以提供任何详细错误的latest.log 或类似文件？
Yes it can, even actually compatible with EpicFight I was suspected. I will sent my log once I'm at leisure today and see what you can find

**Comment by 5A93** [comment]
> 您是否尝试过仅安装 Vanilla Claws（以及任何必要的依赖项）就可以加入这个世界？您是否可以提供任何详细错误的latest.log 或类似文件？

Failed to join a new world and this is latest.log
[Uploading latest.log…]()

**Comment by 5A93** [comment]
[Uploading latest.log…]()

**Comment by 5A93** [comment]
> 您是否尝试过仅安装 Vanilla Claws（以及任何必要的依赖项）就可以加入这个世界？您是否可以提供任何详细错误的latest.log 或类似文件？

[Uploading latest.log…]()

**Comment by 5A93** [comment]
> 您是否尝试过仅安装 Vanilla Claws（以及任何必要的依赖项）就可以加入这个世界？您是否可以提供任何详细错误的latest.log 或类似文件？

[latest.log](https://github.com/DakotaPride/vanilla-claws-forge/files/15165268/latest.log)

**Comment by DakotaPride** [comment]
If you don't mind, try joining with Vanilla Claws installed and let me know if this still occurs. It looks like it may be an issue with Better Combat? I'll have to look further into it first to determine the cause.

**Comment by DakotaPride** [comment]
Something to do with this:
<img width="711" alt="image" src="https://github.com/DakotaPride/vanilla-claws-forge/assets/82760983/39280bf6-9118-49ed-b1e4-39d6fafb5664">

Not sure exactly what "Payload" means in this instance, but whatever mod adds it, might wanna try removing it (as long as it's not important) or report it to the mod author.

**Comment by 5A93** [comment]
> Something to do with this: <img alt="image" width="711" src="https://github.com/DakotaPride/vanilla-claws-forge/assets/82760983/39280bf6-9118-49ed-b1e4-39d6fafb5664">
> 
> Not sure exactly what "Payload" means in this instance, but whatever mod adds it, might wanna try removing it (as long as it's not important) or report it to the mod author.

Seems to be the classic [MC-185901]problem, so I tried to add some patching mod and eventually, it crashed.
[minecraft.log](https://github.com/DakotaPride/vanilla-claws-forge/files/15166567/minecraft.log)

**Comment by 5A93** [comment]
> Something to do with this: <img alt="image" width="711" src="https://github.com/DakotaPride/vanilla-claws-forge/assets/82760983/39280bf6-9118-49ed-b1e4-39d6fafb5664">
> 
> Not sure exactly what "Payload" means in this instance, but whatever mod adds it, might wanna try removing it (as long as it's not important) or report it to the mod author.

Seems to be the classic [MC-185901] problem, added some datapack patching mod and seems that problem is currently solved

**Comment by DakotaPride** [comment]
Okay, let me know if any other issues come up!
